src/test/java/system_tests/TrialSeleniumTest.java

```diff
@@ -0,0 +1,70 @@
+package system_tests;
+
+import org.beanpod.switchboard.entity.DeviceEntity;
+import org.beanpod.switchboard.fixture.DeviceFixture;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static system_tests.HttpHandler.postRequest;
+
+public class TrialSeleniumTest {
+  private static WebDriver driver;
+  private static DeviceEntity deviceEntity;
+
+  @BeforeAll
+  static void setUp() {
+    // TODO set it up for mac/linux:
+    // https://stackoverflow.com/questions/228477/how-do-i-programmatically-determine-operating-system-in-java
+    System.setProperty(
+        "webdriver.chrome.driver", "src\\test\\java\\system_tests\\chromedriver.exe");
+    driver = new ChromeDriver();
+
+    // create a test device
+    deviceEntity = DeviceFixture.getDevice1();
+    HashMap<String, String> deviceParam = new HashMap();
```

Comment:
Small change to avoid a Java warning - do the same for the other spot with new HashMap
```suggestion
    HashMap<String, String> deviceParam = new HashMap<>();
```

Reply:
Done.